src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+import types from '../types/types';
+
+const initialState = {
+  books: [],
+};
+
+// Action creator
+export const checkBookStatus = () => ({
+  type: types.CHECK_BOOK_STATUS,
+});
+
+// Categories reducer
+const categoriesReducer = (state = initialState, action) => {
+  if (action.type === types.CHECK_BOOK_STATUS) {
+    return 'Under construction';
+  }
+  return state;
+};
```

Comment:
**OPTIONAL** - Please use the `switch` expression here, just like you did in the book reducer, when your application gets bigger, using the `if` condition might make your code look less readable, please fix this 😄 😄

Reply:
## Thanks for the feedback @mckent05, I'll implement that right away.